runtime/oti/VMHelpers.hpp

```diff
@@ -2057,13 +2057,20 @@ class VM_VMHelpers
 		return rc;
 	}
 
+#if JAVA_SPEC_VERSION >= 19
+	static VMINLINE bool isMounted(J9VMContinuation *continuation)
```

Comment:
- It is also missing a function description. See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-function-parameters-clearly.
- The name should be made more verbose to avoid ambiguity: `isContinuationMounted`.